ObjectiveGitFramework.xcodeproj/project.pbxproj

```diff
@@ -1364,6 +1381,19 @@
 /* End PBXResourcesBuildPhase section */
 
 /* Begin PBXShellScriptBuildPhase section */
+		5621262D2101D978009A787A /* ShellScript */ = {
+			isa = PBXShellScriptBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			inputPaths = (
+			);
+			outputPaths = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = /usr/bin/python;
+			shellScript = "# encoding=utf-8\n\nimport os, time, codecs\nroot = os.getenv('SRCROOT')\nprint \"update localisation in\", root\nfor (dirpath, dirnames, filenames) in os.walk(root):\n    if \"en.lproj\" in dirnames:\n        os.chdir(dirpath)\n        os.system(\"find ./ -name '*.mm' -o -name '*.m' | xargs genstrings -u -q -o en.lproj\")\n";
```

Comment:
Can you split that into `./script/update-localization` ? That would be ❤️, as that would ~save my soul from Xcode~ make it much easier to edit.